### ISSUE TITLE
Previous Card functionality

### ISSUE THREAD

**Issue by iamsureshraju** [jwasham/computer-science-flash-cards#100]
This project is awesome. Could you please add a "Previous Card" button functionality, so that we can go back on the cards learnt.

**Comment by jwasham** [comment]
You're right, I don't think there's a place to look at the cards you've learned. I honestly don't remember, it's been years since I've looked at this. 
I won't be adding it myself, but if anyone wants to give it a try, I'm open to looking at the pull request.

**Comment by marielledoenges** [comment]
Hi! I will take this on. An open source contribution is required for one of my classes, so I will try to implement this feature.

**Comment by saiteja-yadav** [comment]
Hi! Can you please send the image I'll try to fix that.....

**Comment by marielledoenges** [comment]
Hello, I just made a PR to add in the previous card button. Look it over if you'd like!
